common/app/views/fragments/amp/stylesheets/main.scala.html

```diff
@@ -193,6 +190,27 @@
         margin: 0.1875rem 0.625rem 0.34rem 0;
     }
 
+    @* Dropcap styling *@
+    .drop-cap__inner {
+        float: left;
+        display: inline-block;
+        vertical-align: text-top;
+        font-size: 3rem;
+        line-height: 2.5rem;
+    }
+
+    .drop-cap {
+        float: left;
+        display: inline-block;
+        text-transform: uppercase;
+        -webkit-box-sizing: border-box;
+        -moz-box-sizing: border-box;
+        box-sizing: border-box;
```

Comment:
@sndrs agreed. We're just rushing something out of the door for this week's preview. We'll need to revisit it all, afterwards.